### DISCUSSION TITLE
自定义文件直链窗口在受保护的文件夹内链接超出框

### DISCUSSION THREAD

**Discussion by opoet** [spencerwooo/onedrive-vercel-index#434] [General]
此样式异常仅出现在受保护的文件自定义直连的时候

另外希望能添加点击复制功能 谢谢 `●'◡'●`

![image](https://user-images.githubusercontent.com/30021379/153843611-5879226f-9147-4496-be78-a45f9f61c7fd.png)

**Comment by spencerwooo** [comment]
should be fixed on latest commit ;)

**Reply by opoet** [reply]
未解决，链接还是没有换行显示

![image](https://user-images.githubusercontent.com/30021379/153872189-a0e1bfaa-8ea0-4ec3-bf0e-13f7d12bec02.png)

**Reply by spencerwooo** [reply]
Now the url breaks regardless of preserving whole words - https://tailwindcss.com/docs/word-break#break-all

![Screen Shot 2022-02-14 at 22 43 02-fullpage](https://user-images.githubusercontent.com/32114380/153885335-83ccc4f0-d7f3-485d-9587-40d854adbc5d.png)

**Reply by myl7** [reply]
How about `word-break: break-word;` and fallbacks to `word-break: break-all;`?
Ref: https://stackoverflow.com/questions/12699800/smarter-word-break-in-css https://caniuse.com/word-break#:~:text=Chrome%2C%20Safari%20and%20other%20WebKit/Blink%20browsers%20also%20support%20the%20unofficial%20break%2Dword%20value%20which%20is%20treated%20like%20word%2Dwrap%3A%20break%2Dword

**Comment by spencerwooo** [comment]
复制的话，下面已经有按钮了，还需要吗？没啥意义吧

**Reply by opoet** [reply]
不好意思，我刚开始忽略了这个按钮。我想的是点击链接框就可以自动复制，有这个按钮确实不需要其他的了
